packages/ui/src/components/Menu.svelte

```diff
@@ -14,16 +14,17 @@
 -->
 <script lang="ts">
   import { createEventDispatcher, onDestroy, onMount } from 'svelte'
-  import { generateId, Ref, Doc } from '@hcengineering/core'
+  import { Doc, Ref, generateId } from '@hcengineering/core'
   import ui from '../plugin'
   import { closePopup, showPopup } from '../popups'
-  import { Action } from '../types'
+  import { Action, MenuItem, MenuComponent } from '../types'
   import Icon from './Icon.svelte'
   import Label from './Label.svelte'
   import MouseSpeedTracker from './MouseSpeedTracker.svelte'
   import { resizeObserver } from '../resize'
+  import Component from './Component.svelte'
 
-  export let actions: Action[] = []
+  export let items: MenuItem[] = []
```

Comment:
@haiodo please take a look at these changes

Reply:
Per my point of view, we should keep menus simple and do not allow to render complex components inside, at least at platform level. Maybe you can add custom menu that allows this.

But as a user I don't understand how to use this component, I need to click into this small window, enter a value and then confirm the input somehow... I think that a popup window would be more convenient for this.

Reply:
@aonnikov let's discuss this component with @AlexandrLapin on UX call tomorrow

Reply:
I've reverted the changes. Thanks.